simulator/examples/fill.rs

```diff
@@ -6,14 +6,14 @@ use std::time::Duration;
 
 use embedded_graphics::coord::Coord;
 use embedded_graphics::prelude::*;
-use embedded_graphics::primitives::{Circle, Rect};
+use embedded_graphics::primitives::{Circle, Rect, Triangle};
 
 use simulator::DisplayBuilder;
 
 static CIRCLE_SIZE: i32 = 32;
 
 fn main() {
-    let mut display = DisplayBuilder::new().size(256, 128).scale(2).build();
+    let mut display = DisplayBuilder::new().size(304, 128).scale(2).build();
```

Comment:
Why did this need to change?

Reply:
I added some triangles so I had to increase the window size in order for it to fit all primitives.

Reply:
Ah I see, no problem